.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: cpp
+
+os:
+  # - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
+
+osx_image: xcode9.2
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew tap homebrew/php; brew update; fi
+
+install:
+  # Install packages from Homebrew/Aptitude
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install swig php71 lua; fi
+
+before_script:
+  - mkdir build
+  - cd build
+  - cmake .. -DTINYSPLINE_WITH_RUBY=FALSE -DTINYSPLINE_WITH_PHP=FALSE
+
+script:
+  - make
```

Comment:
Using `cmake --build .` should be less brittle as it handles different build systems for you